docs/contributing/contributing.md

```diff
@@ -1,111 +1,168 @@
 ---
 id: contributing
-title: Contributing 
+title: Contributing XSOAR Content
 ---
 
+This article describes the contribution process for Cortex XSOAR content.  
+Contributing allows clients / partners who made custom content, enhancements to existing content, and bug fixes, to share them with the community by making them publicly available in [Marketplace](/marketplace).  
+Content can be either partner or community supported, and can be either [free or paid](../partners/premium-packs#pricing).
+
+All content (except for paid content packs) is open source, and is hosted on the [Cortex XSOAR Content GitHub Repository](https://github.com/demisto/content), licensed under the [MIT license](https://github.com/demisto/content/blob/master/LICENSE).  
+
+**If you have any questions or need support, feel free to reach out to us on the `#demisto-developers` channel on our [Slack DFIR Community](https://www.demisto.com/community/).**  
 :::note
-This article is focused on contributing Cortex XSOAR Content. If you want to contribute fixes/suggestions to our development and reference Content docs (the site you are currently browsing) go to: [Documentation Contributions](docs-contrib).
+This article is only for content contributions.
+For documentation contributions, refer to [Documentation Contributions](docs-contrib).  
 :::
 
-Thanks for being interested in contributing to Cortex XSOAR. This document describes the Contribution process. If you are not sure whether you should read this, make sure you read the [Getting Started Guide](../concepts/getting-started-guide) first.
+## Content Support Types
+Contributions can be either *officially supported* (by Palo Alto Networks, a Technology Partner, or an individual), or *community supported*.  
 
-Contributing allows you to make the content that you build on Cortex XSOAR available to every client through the  [Marketplace](/marketplace). Content can be either Partner or Community supported, [Free or Paid](../partners/premium-packs#pricing).
+### Officially Supported Packs
+Officially supported packs include an email address or a website of the contributor for customers to contact for support (that they will be referred to in case of an issue), and have a more strict quality-control process.
```

Comment:
```suggestion
Officially supported packs have a more strict quality-control process, and provide an email address or a website of their owner that customers can reach out to (and will be referred to) in case of an issue.
```